### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by IloveJavaa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,20 +28,50 @@ Basic integration First, download the library from npm:
...
+```
+
+3. Run the below commands in Terminal/Cmd.
+```
+$ ionic cap sync
+```
+
+After following all the above steps, Please add the trackier folder in the the `@awesome-cordova-plugins` which is located in the `node_modules`. 
+
+You can downloads the trackier by the click on below link.
```
Comment (Hemant-Mann) [inline README.md]:
```suggestion
You can download the trackier folder by the click on below link.
```

File: README.md [hunk 2]
```diff
@@ -28,20 +28,50 @@ Basic integration First, download the library from npm:
...
 For initialising the Trackier SDk. First, We need to generate the Sdk key from the Trackier MMP panel.
 
-Following below are the steps to retrieve the development key:-
+Following below are the steps to retrieve your SDK key:-
 
-- Login your Trackier Panel
-- Select your application and click on Action button and login as
-- In the Dashboard, Click on the` SDK Integration` option on the left side of panel. 
-- under on the SDK Integration, You will be get the SDK Key.
+1.Login to your Trackier MMP account.
```
Comment (Hemant-Mann) [inline README.md]: New lines not visible in the markdown @IloveJavaa

File: README.md [hunk 3]
```diff
@@ -28,20 +28,50 @@ Basic integration First, download the library from npm:
...
+2.Select the application from dashboard which you want to get the app token for.
+3.Go to SDK Integration via the left side navigation menu.
+4.Copy the SDK Key there to be used as the "app_token".
 
 After follow all steps, Your SDK key look like the below screenshot
```
Comment (Hemant-Mann) [inline README.md]:
```suggestion
After following all the steps, Your SDK key should look like the below screenshot
```

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IloveJavaa) [reply README.md]: Done

**Review by Hemant-Mann — APPROVED** [review]

**Pull request merged**